code/modules/client/preferences.dm

```diff
@@ -998,8 +998,8 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 			if (choice == "Cancel")
 				ShowKeybindings(user)
 				return
-			hotkeys = (choice == "Hotkey")
-			key_bindings = (hotkeys) ? deepCopyList(GLOB.hotkey_keybinding_list_by_key) : deepCopyList(GLOB.classic_keybinding_list_by_key)
+			focus_chat = (choice == "Hotkey")
```

Comment:
If we don't want keybindings to hit focus_chat, this should be removed.

Reply:
It's still preferable for it to be there, as most users of classic keybindings want to combine those.